test/src/test/java/com/distkv/dst/test/supplier/ListRpcTestUtil.java

```diff
@@ -1,9 +1,9 @@
-package org.dst.test.supplier;
+package com.distkv.dst.test.supplier;
 
 
```

Comment:
noblank line

Reply:
I renamed it with IntelliJ IDEA. It generated extra spaces for me. I have manually modified it again.